test/parallel/test-child-process-fork-dgram.js

```diff
@@ -66,13 +78,16 @@ if (process.argv[2] === 'child') {
     sendMessages();
   });
 
+  var iterations = 0;
   var sendMessages = function() {
     var timer = setInterval(function() {
+      iterations++;
+
       client.send(
         msg,
         0,
         msg.length,
-        server.address().port,
+        serverPort,
```

Comment:
Its needed because the server is closed before we have stopped sending. Therefore, you can't call server.address().port. I found this the hard way while building up the changeset.